utils/checks.stanza

```diff
@@ -249,7 +254,7 @@ pcb-check no-connect-pins-not-connected (module:JITXObject) :
     val populated-connected-pins = populated-connected-pins(p)
```

Comment:
You may want to delete this line!

Reply:
Hmm, I see that this is used in the locators. That's ok then!
In this case it is not necessary to call `has-populated-connected-pins?` because you have access to the pins already.

Reply:
I reverted the line